### PR TITLE
drakrun: don't use cmd.exe wrapper when executing target process

### PR DESCRIPTION
**Pull request by icedevml**
depends #318

### CODE REVIEW

**Comment by icedevml** [comment]
needed to check whether Office support still works with this

**Comment by icedevml** [comment]
tested on Win7x64 and Win10 2004, works

TODO test Office

**Comment by icedevml** [comment]
Office tested, works after fixup

File: drakrun/drakrun/main.py [hunk 1]
```diff
@@ -210,22 +213,22 @@ def _get_dll_run_command(pe_data):
...
             elif exports[0][0]:
-                return 'start rundll32 %f,#{}'.format(export[0])
+                return 'rundll32 %f,#{}'.format(export[0])
 
         return None
 
     @staticmethod
     def _get_office_file_run_command(extension, file_path):
-        start_command = ['start']
+        start_command = ['cmd.exe', '/C', 'start']
```
Comment (icedevml) [inline drakrun/drakrun/main.py:231]: looks like `start` is making exhaustive search over `C:\Program Files` (and maybe some other locations too), we should avoid using it whenever possible

File: drakrun/drakrun/main.py [hunk 2]
```diff
@@ -210,22 +213,22 @@ def _get_dll_run_command(pe_data):
 
         for export in exports:
             if export[1] == 'DllRegisterServer':
-                return 'start regsvr32 %f'
+                return 'regsvr32 %f'
```
Comment (icedevml) [inline drakrun/drakrun/main.py:216]: TODO test this path

File: drakrun/drakrun/main.py [hunk 3]
```diff
@@ -210,22 +213,22 @@ def _get_dll_run_command(pe_data):
...
 
             if 'DllMain' in export[1]:
-                return 'start rundll32 %f,{}'.format(export[1])
+                return 'rundll32 %f,{}'.format(export[1])
```
Comment (icedevml) [inline drakrun/drakrun/main.py:219]: TODO test this path

File: drakrun/drakrun/main.py (hunk 1 above)
Comment (icedevml) [reply drakrun/drakrun/main.py:231]: checked, this is reading from registry HKCR

**Review by chivay — APPROVED** [review]

**Pull request merged**